css/style.css

```diff
@@ -23,47 +20,58 @@ nav {
 }
 
 .nav-placeholder {
-    margin: 0 0 10px 0;
+  margin: 0 0 10px 0;
 }
 
 .fixed {
-    position: fixed;
-    top: 0;
-    width: 100%;
-    background-color: rgba(0, 0, 0, 0.5);
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
 }
 
 .fixed .nav-inner {
-    padding: 0 0px;
+  padding: 0 0px;
 }
 
 .fixed .nav-inner-most {
-    width: 100%;
+  width: 100%;
 }
 
-nav ul {
+  nav ul {
     margin: 0;
```

Comment:
also the previous group used 4 spaces a lot, so in the future we will be mindful of editing those indentations before making a PR